src/Components/NavBar/Menus/NewMobileNavMenu.tsx

```diff
@@ -93,32 +155,56 @@ const Menu = ({ isOpen, title, links, showBacknav = true }) => {
   )
 }
 
-const BackLink = () => {
+export const BackLink = () => {
+  const { trackEvent } = useTracking()
   const { pop } = useNavigation()
+  const contextModule = getContextModule()
+
   return (
-    <Box position="absolute" top="-6px">
-      <a
-        href="#"
-        onClick={e => {
-          e.preventDefault()
-          pop()
-        }}
-      >
-        <ChevronIcon
-          direction="left"
-          color={color("black100")}
-          height="10px"
-          width="10px"
-          top="7px"
-          left="5px"
-        />
-      </a>
+    <Box
+      position="absolute"
+      top="-6px"
+      onClick={e => {
+        e.preventDefault()
+        trackEvent({
+          action_type: AnalyticsSchema.ActionType.Click,
+          context_module: contextModule,
+          flow: "Header",
+          subject: "Back link",
+        })
+        pop()
+      }}
+    >
+      <ChevronIcon
+        direction="left"
+        color={color("black100")}
+        height="10px"
+        width="10px"
+        top="7px"
+        left="5px"
+      />
     </Box>
   )
 }
 
+const getContextModule = () => {
```

Comment:
Instead of basing the `contextModule` off of the path, could we instead pass it down through the `MobileSubmenuLink` and access it directly?

Reply:
We talked about this IRL. This is related to the [refactor](https://github.com/artsy/reaction/pull/3238#discussion_r388608576) I made that switched  from passing down `path`, `pop`, and `push` props down through several children props to accesses it where is was to be used to accessing it from the `NavigatorContextProvider` via a local hook. Determining what the context module should be based on the `path` is consistent with that pattern so the conclusion was to keep this as is.